dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/Components/GeneratorWithControl/Controls/ExciterAvrPssOel.mo

```diff
@@ -0,0 +1,155 @@
+    Placement(visible = true, transformation(origin = {-158, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Gain efd_filtGain(k = 10) annotation(
+    Placement(visible = true, transformation(origin = {150, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Math.Add efd_filtAdd(k1 = 1, k2 = -1) annotation(
+    Placement(visible = true, transformation(origin = {118, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.LimIntegrator efd_filtInt(k = 1, outMax = L2, outMin = 0, y_start = Efd0Pu) annotation(
+    Placement(visible = true, transformation(origin = {186, -44}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.RealInput omegaPu annotation(
+    Placement(visible = true, transformation(origin = {-180, -112}, extent = {{-16, -16}, {16, 16}}, rotation = 0), iconTransformation(origin = {-120, -60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
+  Modelica.Blocks.Math.Add dOmega(k1 = -1, k2 = 1) annotation(
+    Placement(visible = true, transformation(origin = {-126, -88}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Sources.Constant OmegaRefPu_const(k = 1) annotation(
+    Placement(visible = true, transformation(origin = {-174, -80}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.LimIntegrator timer(k = 1, outMax = 99, outMin = L1, y_start = L1) annotation(
+    Placement(visible = true, transformation(origin = {-110, 22}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Continuous.TransferFunction leadLag(a = {Tb, 1}, b = {Ta, 1}, initType = Modelica.Blocks.Types.Init.NoInit, x(fixed = true, start = Efd0Pu), y_start = Efd0Pu)  annotation(
+    Placement(visible = true, transformation(origin = {82, -38}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+
+  parameter Modelica.SIunits.PerUnit Efd0Pu(fixed=false);
+  parameter Modelica.SIunits.PerUnit UStator0Pu(fixed=false);
+  parameter Modelica.SIunits.PerUnit If0Pu(fixed=false);
+  final parameter Modelica.SIunits.PerUnit UsRef0Pu = Efd0Pu / G + UStator0Pu;
+
+equation
+  connect(leadLag.y, efd_filtAdd.u1) annotation(
+    Line(points = {{94, -38}, {104, -38}, {104, -38}, {106, -38}}, color = {0, 0, 127}));
+  connect(kMulDU.y, leadLag.u) annotation(
+    Line(points = {{60, -38}, {68, -38}, {68, -38}, {70, -38}}, color = {0, 0, 127}));
+  connect(dOmega_filt3.y, dOmega_filt3_minMax.u) annotation(
+    Line(points = {{-8, -88}, {0, -88}, {0, -88}, {0, -88}}, color = {0, 0, 127}));
+  connect(dOmega_filt2.y, dOmega_filt3.u) annotation(
+    Line(points = {{-44, -88}, {-32, -88}, {-32, -88}, {-32, -88}}, color = {0, 0, 127}));
+  connect(dOmega_filt1.y, dOmega_filt2.u) annotation(
+    Line(points = {{-80, -88}, {-68, -88}, {-68, -88}, {-68, -88}}, color = {0, 0, 127}));
+//connect(efdPu, efdPuPin.value) annotation(
+//  Line);
+  efdPu = efdPuPin.value;
```

Comment:
The equation could be put before or after all the connect

Reply:
resolved, see [e1000c7](https://github.com/dynawo/dynawo/pull/2394/commits/e1000c7adff25e271a2e2214bb7197d005b00633)